static/src/javascripts/projects/common/modules/analytics/tech-feedback.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'bean',
+    'fastdom',
+    'common/utils/$'
+], function (
+    bean,
+    fastdom,
+    $
+) {
+
+    return {
+        init: function () {
```

Comment:
Looks good, I'd quickly try

```js
$('.js-tech-feedback').on('submit', function () {
   var oldHref = this.attr('action');
   this.attr('action', oldHref + '&width=' + window.innerWidth);
})
```

or something along these lines.
This way you don't have to touch the DOM on page load, but only when the user click that button.

Otherwise :+1: